docs/content/direct/core-chart-argocd.md

```diff
@@ -1,16 +1,34 @@
 # Using Argo CD with KubeStellar Core chart
 
+## Table of Contents
+- [Overview](#overview)
+- [Pre-requisites](#pre-requisites)
+- [Installing Argo CD using KubeStellar Core chart](#installing-argo-cd-using-kubestellar-core-chart)
+- [Deploying Argo CD applications](#deploying-argo-cd-applications)
+
+## Overview
+
 This documents explains how to use the KubeStellar core Helm chart to:
 
 - deploy Argo CD in KubeFlex hosting cluster;
 - register every WDS as a target cluster in Argo CD; and
 - create Argo CD applications as specified by the chart values.
 
+For a detailed step-by-step installation guide with expected outputs, see [Step-by-Step Installation Guide](core-chart.md).
+
 ## Pre-requisites
 
-The prerequisites are the same as [installing KubeStellar using the Core chart](core-chart.md#pre-requisites).
+Before installing Argo CD with KubeStellar Core chart, ensure you have:
+
+- All prerequisites from [installing KubeStellar using the Core chart](core-chart.md#pre-requisites)
+- A properly configured KubeFlex hosting cluster
+- Helm installed and configured
+- kubectl access to your cluster
 
-The settings described in this document are an extesnion of the KubeStellar Core chart settings described [here](core-chart.md#kubestellar-core-chart-values).
+The settings described in this document are an extension of the KubeStellar Core chart settings described [here](core-chart.md#kubestellar-core-chart-values).
+
+> The KubeStellar core chart can optionally be used to install Argo CD in the KubeFlex hosting cluster and register each KubeStellar WDS as an Argo CD target cluster. The core chart also has the option to define some Argo CD Applications.  
+> This section will cover installing Argo CD and mapping WDSes to target clusters; the next section will show how to also define some Applications.

```

Comment:
This text does not belong under the section heading "Pre-requisites", the subject is different.